Modules/Remote/CudaCommon.remote.cmake

```diff
@@ -0,0 +1,45 @@
+#-- # Grading Level Criteria
+#-- Contact: Simon Rit <simon.rit@creatis.insa-lyon.fr>
+#-- EVALUATION DATE: 2021-11-03
+#-- EVALUATORS: [Simon Rit]
+#--
+#-- ## Compliance level 5 star (AKA ITK main modules, or remote modules that could become core modules)
+#--   - [ ] Widespread community dependance
+#--   - [ ] Above 90% code coverage
+#--   - [ ] CI dashboards and testing monitored rigorously
+#--   - [ ] All requirements of Levels 4,3,2,1
+#--
+#-- ## Compliance Level 4 star (Very high-quality code, perhaps small community dependance)
+#--   - [x] Meets all ITK code style standards
+#--   - [x] No external requirements beyond those needed by ITK proper
```

Comment:
I think this should be off, because CUDA is an external requirement not needed by main ITK library.

Reply:
Of course, sorry, I overlooked this.